iroh/src/lib.rs

```diff
@@ -0,0 +1,8 @@
+pub mod api;
+mod clientapi;
+
+pub use crate::clientapi::ClientApi as Api;
+pub use bytes::Bytes;
+pub use cid::Cid;
+pub use libp2p::gossipsub::MessageId;
+pub use libp2p::{Multiaddr, PeerId};
```

Comment:
Making the client api depend on libp2p is problematic because of how much dependencies libp2p comes with. Something leaner and more self contained would be better.

Reply:
I've brought this up to @dignifiedquire